source/redux/wallet.js

```diff
@@ -22,9 +22,14 @@ export const walletSlice = createSlice({
       state.emoji = emoji;
     },
     setAccountInfo: (state, action) => {
+      if (action.payload === undefined) return;
```

Comment:
is this check for when it's specifically undefined? else you can do `if (!action.payload)`

Reply:
Just saw the comment, yeah, It looks cleaner, will change it on a future pr for fixing more warnings